Packages/cdms2/Lib/dataset.py

```diff
@@ -283,9 +283,10 @@ def openDataset(uri,mode='r',template=None,dods=1,dpath=None, hostObj=None):
     uri = string.strip(uri)
     (scheme,netloc,path,parameters,query,fragment)=urlparse.urlparse(uri)
     if scheme in ('','file'):
+        if uri[:7].lower() == "file://":
```

Comment:
@aashish24 first off it's cdms open not python open.
Second we are using:
`(scheme,netloc,path,parameters,query,fragment)=urlparse.urlparse(uri)`
but if it starts with `file://` then `path` and `root` are both empty strings.